### PR TITLE
loader: Add AFR docs to README.md

### PR DESCRIPTION
**Pull request by DLehenbauer**
Expanding documentation for 'webpack-fluid-loader' to include information on connecting to AFR.

### CODE REVIEW

File: packages/tools/webpack-fluid-loader/README.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ This folder contains the webpack-fluid-loader. This package is meant to be used
 
 This loader is intended for development purposes only and should not be used in production.
 
-The following environment variables can be defined when running `webpack --serve` to change the behavior of webpack-fluid-loader:
+The following environment variables can be defined when running `webpack --serve` to change the behavior of webpack-fluid-loader.
+
+See the [Azure Fluid Relay](#azure-fluid-relay) and [Sharepoint](#sharepoint) sections below for information on connecting to these services.
```
Comment (tylerbutler) [inline packages/tools/webpack-fluid-loader/README.md:9]:
```suggestion
See the [Azure Fluid Relay](#azure-fluid-relay) and [SharePoint](#sharepoint) sections below for information on connecting to these services.
```

File: packages/tools/webpack-fluid-loader/README.md [hunk 2]
```diff
@@ -75,9 +77,34 @@ or in an optional `config.json` file in the `baseDir` passed into `webpack-fluid
 
 ```
 
+## Azure Fluid Relay
+To use Azure Fluid Relay, [provision a Fluid Relay instance](https://learn.microsoft.com/en-us/azure/azure-fluid-relay/how-tos/provision-fluid-azure-portal) in the Azure Portal.
+To select the deployment region, consult [the availability table](https://azure.microsoft.com/en-us/explore/global-infrastructure/products-by-region/?products=fluid-relay) for the Azure Fluid Relay product<sup id="a1">[1](#f1)</sup>.
```
Comment (tylerbutler) [inline packages/tools/webpack-fluid-loader/README.md:82]: I _think_ you can remove the en-us from the URLs and folks will be redirected to a localized page (if one's available).

**Review by tylerbutler — APPROVED** [review]
📝

File: packages/tools/webpack-fluid-loader/README.md (hunk 1 above)
Comment (DLehenbauer) [reply packages/tools/webpack-fluid-loader/README.md:9]: Oops.  Thanks for noticing!

File: packages/tools/webpack-fluid-loader/README.md (hunk 2 above)
Comment (DLehenbauer) [reply packages/tools/webpack-fluid-loader/README.md:82]: Seems to work.  Nice suggestion!

**Pull request merged**